packages/fhir-converter/src/templates/cda/Sections/Hospital_Discharge_Medications.hbs

```diff
@@ -5,7 +5,11 @@
                     {{>Resources/Medication.hbs medication=medAdm.substanceAdministration.consumable.manufacturedProduct.manufacturedMaterial ID=(generateUUID (toJsonString medAdm.substanceAdministration.consumable.manufacturedProduct.manufacturedMaterial))}},
                     {{>References/MedicationAdministration/medicationReference.hbs ID=(generateUUID (toJsonString medAdm.substanceAdministration)) REF=(concat 'Medication/' (generateUUID (toJsonString medAdm.substanceAdministration.consumable.manufacturedProduct.manufacturedMaterial)))}},
 
-                    {{>Resources/Organization.hbs org=medAdm.substanceAdministration.performer.assignedEntity.representedOrganization ID=(generateUUID (toJsonString medAdm.substanceAdministration.performer.assignedEntity.representedOrganization))}},
```

Comment:
generation dangling orgs because 
a) no check that org is not undefined 
b) no references to it in Medication Administration